python/Ganga/test/GPI/Internals/TestRepo.py

```diff
@@ -16,7 +16,10 @@ def __init__(self, name):
     def _read_access(self, root, obj):
         self.repo.load([root._registry_id])
 
-    def _write_access(self, root):
+    def _acquire_session_lock(self, root):
```

Comment:
Almost certainly true, but again, I feel that's out of the scope of this PR at present. I really just wanted to get the basics done of removing `_getReadAccess` and `_getWriteAccess` and then after we know what's actually happening, it makes it a lot easier to do these kind of changes.